### PR TITLE
Add saas files and saas deploy jobs on service page 

### PR DESCRIPTION
**Pull request by violetyao**
Jira ticket: https://issues.redhat.com/browse/APPSRE-2953

### CODE REVIEW

File: src/pages/elements/Service.js [hunk 1]
```diff
@@ -247,6 +322,8 @@ function Service({ service, reports, documents }) {
           </details>
         </div>}
 
+      {<SaasFiles saas_files={ saas_files.filter(file => file['app']['name'] === service['name'])} settings={settings} />}
```
Comment (BumbleFeng) [inline src/pages/elements/Service.js:332]: You can reuse this https://github.com/app-sre/visual-qontract/blob/master/src/pages/elements/Service.js#L54-L61

Comment (violetyao) [reply src/pages/elements/Service.js]: Updated!

File: src/pages/elements/Service.js [hunk 2]
```diff
@@ -40,16 +50,86 @@ const EscalationPolicy = ({app}) => {
...
+const SaasDeployJobs = ({f, settings}) => {
+  const job_template_name = settings[0].saasDeployJobTemplate;
+  var job_lst = [];
+  var job_name_lst = [];
+  var job_name;
+  for (var template of f.resourceTemplates) {
+    for (var target of template.targets) {
+      job_name = job_template_name + '-' + f.name + '-' + target.namespace.environment.name;
+      if (!job_name_lst.includes(job_name)) {
+        job_lst.push(<li><a href={f.instance.serverUrl + "/job/" + job_name + "/api/json?tree=builds[timestamp,result]"}> {job_name} </a></li>);
```
Comment (BumbleFeng) [inline src/pages/elements/Service.js:63]: We can use the direct job url here, no need for using api url

Comment (violetyao) [reply src/pages/elements/Service.js]: Gotcha! It makes sense. Updated!

**Review by BumbleFeng — APPROVED** [review]

**Pull request merged**